File: Module 0 - Programming Fundamentals/Operators/Challenge Problem/BasicCalculator.cpp
//Build a simple calculator supporting +, -, *, /, and %.

#include<iostream>
using namespace std;

int main(){
    int a ,b;
    cout << "Enter First number :- ";
    cin >> a;
    cout << "Enter Second Number :- ";
    cin >> b;
    
    cout << a << '+' << b << ":- "<< a+b << endl;
    cout << a << '-' << b << ":- "<< a-b << endl;
    cout << a << '*' << b << ":- "<< a*b << endl;
    cout << a << '/' << b << ":- "<< a/b << endl;
    cout << a << '%' << b << ":- "<< a%b << endl;
    
    return 0;
}

File: Module 0 - Programming Fundamentals/Operators/Challenge Problem/BasicCalculator.py
# Build a simple calculator supporting +, -, *, /, and %.

a = float(input("Enter first number: "))
b = float(input("Enter second number: "))

print(a,"+",b,":- ",a+b)
print(a,"-",b,":- ",a-b)
print(a,"*",b,":- ",a*b)
if b != 0:
    print(a,"/",b,":- ",a/b)
    print(a,"%",b,":- ",a%b)